### DISCUSSION TITLE
Better Categories on Home Screen

### DISCUSSION THREAD

**Discussion by memetrusidovski** [jellyfin/jellyfin#10779] [General]
I'm wondering if there is a plugin or mod that can create better categories for my media automatically? I just installed Jellyseerr yesterday and I basically want to emulate that homescreen but with my media. I'm looking around and I can't find anything. I was wondering if anyone's seen something like that or how hard it would be to make something like that. 
Basically I want to have categories beyond the standard "movies" "tv shows" "anime" etc.
I want to do one like 
"Platforms"
       ->"Netflix" "Disney Plus" "Prime Video" etc
"Genres" 
        ->"Comedy" "Action" "Animated"
"Kids"
        ->"Idk whatever my little cousins watch"
"Seasonal"
        "Christmas Movies" "Halloween" etc. 

and so on. I'm not a C# dev but im pretty good with Java so im sure i can figure it out. I've never worked with making Jellyfin plugins so id appreciate if someone could let me know if this is a waste of time or something doable and useful.

Thanks!

**Comment by memetrusidovski** [comment]
oh and also I forgot. I wanted to possibly include a recommendation engine or at least a popular movies, anime, and tv section if it's not possible. I built a recommendation engine with Gorse a few months ago but i dont think a single user would have enough data to make this useful. so it may have to be an api or something.

**Reply by Cpyrighted** [reply]
I know this is months later but you could potentially create a recommendation engine using trakt's api, they collect user data from thousands of people around the world (with complete consent from the users) it is the entire services purpose for Movies and TV Shows.

They have trending, popular, featured as well as genres and they track what content is on what streaming services, they even have a upcoming section, you should be able to pull all this data from their API with ease.

Trakt's API is on Github: https://github.com/trakt/api-help

For Anime, I recommend using AniList's api they basically do the same thing as trakt except they purely focus on Anime only.

Using both of these you should be able to achieve your goals, for seasonal sections though while it may be possible to do this using these api's I'd not do so, it might be more worth and recommended to create a curated list of seasonal content that you rate highly, generic curated lists are well... generic. The best lists are often the ones where the curator has taken the time to watch and evaluate each piece of content, the other seasonal lists based off what the userbase watched are not generally the best holiday movies/shows but simple the ones people watched, leading to a lot of meh or filler content that we all might watch once but wouldn't choose to watch again.

**Reply by HammyHavoc** [reply]
Personally, I would separate the categories plugin from a recommendations plugin (applying UNIX philosophy with modular functionality rather than monolithic).
